nornir/core/connections.py

```diff
@@ -21,6 +21,7 @@ class ConnectionPlugin(ABC):
     """
 
     __slots__ = ("connection", "state")
+    default_conn_name = ""
```

Comment:
I really like this change, not sure if this could be an abstract property though to make sure it's always implemented though.

Reply:
Also, not sure if we should just rename this to `plugin_name`. So in the code it'd look like `Paramiko.plugin_name`. In the end, it's not the "plugin name" what you change when you use something like `netmiko_telnet`, the name remains the same, you just look for a different place for the connection options.

Reply:
I think we should change it to `plugin_name`, I see your point. Or may be just `name`?
I will add this as an abstract property.